### PR TITLE
Fixed some pytest compatibility issues

### LINKED ISSUE #73: Improve coverage
See https://github.com/diffpy/diffpy.pdfmorph/pull/67#pullrequestreview-1338519151

### PR DESCRIPTION
**Pull request by Sparks29032**
Fix #73

### CODE REVIEW

**Comment by sbillinge** [comment]
This passed all tests on the CI.  Do you want me to merge it?

**Comment by Sparks29032** [comment]
There are two more errors I want to fix before merging. I will push another commit on the same branch.

**Comment by sbillinge** [comment]
> There are two more errors I want to fix before merging. I will push another commit on the same branch.

👍

File: diffpy/pdfmorph/pdfmorphapp.py [hunk 1]
```diff
@@ -190,19 +190,23 @@ def custom_error(self, msg):
     group.add_option(
         '--mag', type="float", help="Magnify plot curves beyond MAGLIM by MAG."
     )
+    group.add_option(
+        '--lwidth', type="float", help="Line thickness of plotted curves."
```
Comment (Sparks29032) [inline diffpy/pdfmorph/pdfmorphapp.py:194]: added option for visibility adjustments, can remove if unnecessary

File: diffpy/pdfmorph/pdfmorphapp.py [hunk 2]
```diff
@@ -253,31 +258,39 @@ def main():
         if opts.baselineslope is None:
             refpars.append("baselineslope")
             config["baselineslope"] = -0.5
+
+    def nn_value(val, name):
```
Comment (sbillinge) [inline diffpy/pdfmorph/pdfmorphapp.py:262]: is there a reason you are defining this inside main?  if not, pull it out for greater readability

**Review by sbillinge — COMMENTED** [review]
please see my inline comments.  They are kind of trivial so I am going to merge this as I have to leave now and I want you to be able to progress, but please read the comments and if any changres are needed you ca ndo them on another pr.

File: diffpy/pdfmorph/pdfmorphapp.py (hunk 2 above)
Comment (Sparks29032) [reply diffpy/pdfmorph/pdfmorphapp.py:262]: will do

**Pull request merged**